ledger/ledger-on-sql/src/main/scala/com/daml/ledger/on/sql/package.scala

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2020 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.on
```

Comment:
`on` is a quite generic term. From a cursory look it doesn't appear to be a reserved keyword in neither Java nor Scala, so I guess it's unlikely to be an issue, but I would consider dropping it or replacing it. `com.daml.ledger.sql` seems equally telling.

Reply:
That is fair; happy to rename this along with renaming the Bazel package at a later point (after we decide as a group).